README.md

```diff
@@ -1,48 +1,7 @@
-# curriculum-databases-projects-template
-
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
-
-
-## Getting Started
-
-This repository includes files with plain SQL that can be used to recreate a database:
-
-- Use [schema.sql](./schema.sql) to create all tables.
-- Use [data.sql](./data.sql) to populate tables with sample data.
-- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
-
-
-## Authors
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
-
-Give a ⭐️ if you like this project!
-
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
+The project was aiming to create a mobile skeleton 
+x-special/nautilus-clipboard
+copy
+file:///home/isaka/Pictures/Screenshot%20from%202021-06-05%2010-15-21.png
+![Screenshot from 2021-06-05 10-15-21](https://user-images.githubusercontent.com/75973193/120885626-95bed080-c5f2-11eb-977a-d4e6475f142a.png)
+Made from pure html and css
+With the use of flexbox and alittle bit of box model properties
```

Comment:
This issue raised by the previous TSE still persists. Please check the template and try to improve your readme file.

- [ ] The project should have a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).

 
     * A good README should have a description that is clear, short, and to the point.
 
     * It should describe the importance of your project, what it does and how to use it. This way technical and non-technical users can understand how to use it.  Refer to this template [here](https://github.com/microverseinc/readme-template) and customize it to your needs.